### PR TITLE
Clarify more config is required

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/10424

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws

File: skype/skype-ps/skype/New-CsOnlineSchedule.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ The New-CsOnlineSchedule cmdlet creates a new schedule for the Auto Attendant (A
 - For a fixed schedule, at most 10 date-time ranges can be specified.
 - You can create a new date-time range for a fixed schedule by using the New-CsOnlineDateTimeRange cmdlet.
 - The return type of this cmdlet composes a member for the underlying type/implementation. For example, in case of the weekly recurrent schedule, you can modify Monday's time ranges through the Schedule.WeeklyRecurrentSchedule.MondayHours property. Similarly, date-time ranges of a fixed schedule can be modified by using the Schedule.FixedSchedule.DateTimeRanges property.
+- Schedules can then be used by [New-CsAutoAttendantCallHandlingAssociation](New-CsAutoAttendantCallHandlingAssociation.md).
```
[the remark this answers is not in the document]
Comment (get-itips) [reply skype/skype-ps/skype/New-CsOnlineSchedule.md:44]: Can't commit as it says no changes were made @JohanFreelancer9

[the remark this answers is not in the document]
Comment (JohanFreelancer9) [reply skype/skype-ps/skype/New-CsOnlineSchedule.md:44]: @get-itips, my apologies, I've deleted it

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**